packages/view-types/statistical-plots/src/ExpressionHistogram.js

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import clamp from 'lodash/clamp';
 import { VegaPlot, VEGA_THEMES } from '@vitessce/vega';
+import { debounce } from 'lodash';
```

Comment:
I am using debounce, so that new cell set is created only after the user has finished selection. The assumption here is that the user has finished the selection when there's been no mouse movement on the expression histogram for a second.

I initially tried to implement this with vega-lite, unfortunately it is not possible, because vega-lite supports very limited type of signals. I had to either migrate the spec to be in vega, or use this.

Reply:
Good point. I have added a comment and links.